test/JDBC/expected/BABEL-4281.out

```diff
@@ -0,0 +1,104 @@
+BEGIN TRAN BABEL4281_T1; 
+GO
+
+CREATE TABLE t_babel4281 (a int, b int);
+GO
+
+-- The third parameter is true to set config back to default after transaction is committed
+select set_config('parallel_setup_cost', 0, true);
+select set_config('parallel_tuple_cost', 0, true);
+select set_config('min_parallel_table_scan_size', 0, true);
+GO
+~~START~~
+text
+0
+~~END~~
+
+~~START~~
+text
+0
+~~END~~
+
+~~START~~
+text
+0
+~~END~~
+
+
+
+-- show explicitly this is a parallel query plan
+select set_config('babelfishpg_tsql.explain_timing', 'off', false);
+GO
+~~START~~
+text
+off
+~~END~~
+
+
+select set_config('babelfishpg_tsql.explain_summary', 'off', false);
+GO
+~~START~~
+text
+off
+~~END~~
+
+
+SET BABELFISH_STATISTICS PROFILE ON
+GO
+
+select a, count(*) from t_babel4281 group by a order by 2; -- should not crash
+GO
+~~START~~
+int#!#int
+~~END~~
+
+~~START~~
+text
+Query Text: select a, count(*) from t_babel4281 group by a order by 2
+Sort  (cost=38.27..38.77 rows=200 width=8) (actual rows=0 loops=1)
```

Comment:
Last time I forgot to mention : 
```
select set_config('babelfishpg_tsql.explain_costs', 'off', false)
```
we should turn off the explain cost in this case, to avoid flaky test cases

Reply:
Updated the test